manifests/cluster-rbac/argocd-application-controller-clusterrolebinding.yaml

```diff
@@ -13,4 +13,3 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: argocd-application-controller
```

Comment:
I'm pretty sure this is wrong. ClusterRoles (and Roles) must be bound to ServiceAccounts. Since there is no ClusterServiceAccount, you must specify a namespace of the ServiceAccount. I think this is optional for RoleBindings, but mandatory for ClusterRoleBindings

Reply:
I never heard about ClusterServiceAccount, guess there is no such thing.

The point of removing the `namespace` directive from `subjects` of both RoleBindings and ClusterRoleBindings is the same: you don't know a namespace of service accounts if you don't specify it in specs.

Namespace came from `-n` flag in `kubectl apply` command ([Install Argo CD](https://argoproj.github.io/argo-cd/getting_started/#1-install-argo-cd)), or from `namespace:` directive in [kustomization.yaml](https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#manage-argo-cd-using-argo-cd).

Right now I can't deploy to any namespace except argocd, because ClusterRoleBinding nails it down

Reply:
@jessesuen I am also facing the same issue. By adding `namespace: argocd` here we are restricted to deploying to argocd. I am not able to use kustomize to generate a proper deployment yaml for another namespace as this value don't change. 

If we remove it and build using kustomize by providing a namespace, proper value will be added here. 

If we remove this and people try to use `install.yaml` directly it will be an issue. Because the current `install.yaml` looks like to be generated without adding any namespace. I believe this is why we are forced to hardcode the namespace here. I think we should remove this values and generate the `install.yaml` by using `argocd` in the kustomize namespace parameter.